library/src/main/scala/scala/scalajs/js/Object.scala

```diff
@@ -202,6 +220,38 @@ object Object extends Object {
    */
   def preventExtensions(o: Object): o.type = native
 
+  /** <span class="badge badge-ecma2015" style="float: right;">ECMAScript 2015</span>
+   *
+   *  Object.is() determines whether two values are the same value. Two values
+   *  are the same if one of the following holds:
+   *
+   *  <ul>
+   *    <li>both undefined
+   *    <li>both null
+   *    <li>both true or both false
+   *    <li>both strings of the same length with the same characters in the same
+   *        order
+   *    <li>both the same object (means both object have same reference)
+   *    <li>both numbers and
+   *      <ul>
+   *        <li>both +0
+   *        <li>both -0
+   *        <li>both NaN
+   *        <li>or both non-zero and both not NaN and both have the same value
+   *      </ul>
+   *    </li>
+   *  </ul>
+   *
+   *  This is not the same as being equal according to JavaScript's `===`
+   *  operator (exposed as
+   *  [[scala.scalajs.js.special.strictEquals js.special.strictEquals]] in
```

Comment:
I don't know why but Scaladoc unfortunately says

> Could not find any member to link for "scala.scalajs.js.special.strictEquals".

If you want to fiddle with it until it works, be my guest. Otherwise just use

    `js.special.strictEquals`

instead (without link).

Reply:
Attempted to fix in https://github.com/scala-js/scala-js/pull/3869/commits/a971cbc4acac5ff2e92f9f66d357c162a1164590